source/opt/dataflow.h

```diff
@@ -87,18 +89,23 @@ class DataFlowAnalysis {
   // worklist until all fixed.
   VisitResult RunOnce(Function* function, bool is_first_iteration);
 
-  friend struct WorklistComparator;
-  struct WorklistComparator {
-    DataFlowAnalysis& analysis;
+  friend class WorklistComparator;
```

Comment:
Do we need the friend declaration? Quick search in the C++11 standard gave me:
> standard $11.7.1
> "A nested class is a member and as such has the same access rights as any other member. The members of an enclosing class have no special access to members of a nested class; the usual access rules shall be obeyed"

Reply:
Apparently this was changed since C++03! Wasn't aware of that, thanks. Will change.